cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/common/Tweet.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.templates.etl.common;
+
+/**
+ * POJO that represents a Tweet.
+ */
+public class Tweet {
```

Comment:
Can include the following in the tweet for it to be useful. The information is all available in the twitter4j status object. http://twitter4j.org/javadoc/twitter4j/Status.html

* Created Date
* Favorite Count
* Lat and Long of Geo Location if available
* Retweet Count
* Source &
* Is Retweet